ui/src/pages/backoffice/Acquisition/Order/OrderDetails/index.js

```diff
@@ -0,0 +1,20 @@
+import { connect } from 'react-redux';
+
+import { fetchOrderDetails } from './state/actions';
+import OrderDetailsComponent from './OrderDetails';
+
+const mapStateToProps = state => ({
```

Comment:
do you need redux in this one? maybe I missed some components but in general it looks to me that you are passing data down as prop in the most of them

Reply:
You are right, we don't need redux. I'll remove it.